src/renderers/dom/shared/ReactDOMComponent.js

```diff
@@ -263,9 +263,11 @@ function assertValidProps(component, props) {
   }
   invariant(
     props.style == null || typeof props.style === 'object',
-    'The `style` prop expects a mapping from style properties to values, ' +
+    ' A component inside its render method has incorrect inline style syntax.' +
```

Comment:
You still have a leading space inside the text which doesn't belong. But do we really need this new sentence? I don't think it really adds anything and with the addendum we'll say to check the render method of a component.